plugins/outputs/exec/README.md

```diff
@@ -38,6 +38,10 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Timeout for command to complete.
   # timeout = "5s"
 
+  ## Whether the command gets executed once per metric, or once per metric batch
+  ## The serializer will also run in batch mode when this is true.
+  # use_batch_format = true
```

Comment:
Maybe rename to `batch_execution`? I'm not sure, others may give their opinion as well..

Reply:
Nevermind, I saw @powersj advised the other way round..

Reply:
I had `batch_exec` [at first](https://github.com/influxdata/telegraf/pull/13672#discussion_r1275281226), but changed it to be consistent with other plugins. I'm now thinking that removing the functionality of running the command once per metric is a good idea. The `use_batch_format` name then makes much more sense, as it isn't also controlling how the program gets executed. It also would address your concern on what error to return above (because there could only ever be a single error.) Then `use_batch_format = false` along with `metric_batch_size = 1` would emulate the current behavior of this PR when batched mode is off.

Reply:
This can be marked as resolved..